### DISCUSSION TITLE
ALT field filling via AI bot

### DISCUSSION THREAD

**Discussion by zero0n3** [tuskyapp/Tusky#4756] [Ideas]
Hello
would it be possible to make the ALT field of the images automatically filled in by an AI bot, during the image upload phase on the client, before publishing?
the ALT field is designed to be filled in "brutally" with the description of the image, without any human imagination. It is a mere description of the photo. And the latest AI solutions, based on the tests done, perfectly cope with this simple task.
I therefore wonder if it were possible to make sure that during the upload phase of the photos, these are also checked in real time and described directly by a bot.

**Comment by connyduck** [comment]
Of course it is possible. 
But I disagree with your claim that alt text is just a mechanical description, it can and should add important context and highlight what is important for the uploader, a bot can't know that. It can even be [approached like writing poetry](https://alt-text-as-poetry.net/).
Combined with the fact that Tusky aims to be a small and leightweight client it is very unlikely that we will add a feature like that.
